apps/src/p5lab/P5Lab.js

```diff
@@ -232,6 +232,10 @@ P5Lab.prototype.init = function(config) {
     );
   }
   this.level = config.level;
+  if (this.level.validationStrings) {
+    this.level.validationStrings = JSON.parse(this.level.validationStrings);
```

Comment:
`level.validation_strings` is passed down as json so it needs to be parsed somewhere. Whether it's here or somewhere else isn't really important.